projects/plugins/boost/app/modules/optimizations/minify/Minify.php

```diff
@@ -24,49 +21,33 @@ public function setup() {
 			return;
 		}
 
-		add_action( Config::get_cron_cache_cleanup_hook(), 'jetpack_boost_page_optimize_cache_cleanup' );
-		register_deactivation_hook( JETPACK_BOOST_PATH, 'jetpack_boost_page_optimize_deactivate' );
-		register_uninstall_hook( JETPACK_BOOST_PATH, 'jetpack_boost_page_optimize_uninstall' );
+		if ( ! self::$called ) {
+			add_action( Config::get_cron_cache_cleanup_hook(), 'jetpack_boost_page_optimize_cache_cleanup' );
+			register_deactivation_hook( JETPACK_BOOST_PATH, 'jetpack_boost_page_optimize_deactivate' );
+			register_uninstall_hook( JETPACK_BOOST_PATH, 'jetpack_boost_page_optimize_uninstall' );
 
-		jetpack_boost_page_optimize_schedule_cache_cleanup();
+			jetpack_boost_page_optimize_schedule_cache_cleanup();
+		}
 
 		add_action( 'init', array( $this, 'init_concatenate' ) );
 
-		// Disable Jetpack photon-cdn for static JS/CSS
-		add_filter( 'jetpack_force_disable_site_accelerator', '__return_true' );
-	}
+		if ( ! self::$called ) {
+			// Disable Jetpack photon-cdn for static JS/CSS
+			add_filter( 'jetpack_force_disable_site_accelerator', '__return_true' );
+		}
 
-	public static function get_slug() {
-		return 'minify';
+		self::$called = true;
 	}
 
 	public static function is_available() {
-		return defined( 'JETPACK_BOOST_MINIFY' ) && true === JETPACK_BOOST_MINIFY;
+		return true;
 	}
 
 	public function setup_trigger() {
 		return 'init';
 	}
 
 	public function init_concatenate() {
```

Comment:
I think these methods need docblocks explaining their purpose.

Reply:
Thanks 😅  Those were removed in 286067045c360986184d090cb638bc9956fd7bcf, but I've added some comments in e07c9316040f1b0796fefecade6a677ad9ce432c.